### PR TITLE
Covid?!

### PR DESCRIPTION
**Pull request by toxic-devil**
Fixed Spelling Error & Fixed Country name With Args + ( Defined ) ✅️

### CODE REVIEW

**Comment by TOXIC-DEVIL** [comment]
Emoji ×   

;-)

File: plugins/covid.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 let fetch = require('node-fetch')
-let handler = async (m, { text }) => {
+let handler = async (m, { text, args }) => {
   let res = await fetch(global.API('https://covid19.mathdro.id', '/api/countries/'+ (text)))
```
Comment (Ftwrr) [inline plugins/covid.js:3]: :/

Comment (TOXIC-DEVIL) [reply plugins/covid.js:3]: What 🙂

Comment (TOXIC-DEVIL) [reply plugins/covid.js:3]: Did you mean this define?! 🙂

Comment (TOXIC-DEVIL) [reply plugins/covid.js]: Is this now resolved?!

**Pull request closed without merging**